packages/react-components/src/utils/__tests__/common.test.tsx

```diff
@@ -97,3 +98,17 @@ describe('isLink', () => {
     expect(isLink('https://example.com')).toBeTruthy();
   });
 });
+
+describe('equals', () => {
+  const testArray = ['Team ASAP', 'Team Chen', 'Team Allesi'];
+
+  it('returns true when both array contain the same elements in the same order', () => {
+    expect(equals(testArray, testArray.slice())).toBeTruthy();
+  });
+  it('returns false when both arrays have the same values in diff order', () => {
+    expect(equals(testArray, testArray.slice().reverse())).toBeFalsy();
+  });
+  it('returns false when arrays differ', () => {
+    expect(equals(testArray, testArray.slice(0, 1))).toBeFalsy();
+  });
+});
```

Comment:
Why do we need this, Jest should be able to do this for us.

Reply:
Done